framework/gltf_loader.cpp

```diff
@@ -627,7 +631,9 @@ sg::Scene GLTFLoader::load_scene(int scene_index)
 				vkb::sg::Texture *tex = textures.at(gltf_value.second.TextureIndex());
 
 				if (texture_needs_srgb_colorspace(gltf_value.first))
+				{
```

Comment:
Applying the brace to the single line code can be optional?

Reply:
Warned against by clang tidy

Reply:
I thought this was a setting in clang-tidy to accept style guides (i.e. can accept Google style guide as an example, there's obviously several others).  Don't have to use KnR for brace placement.

Reply:
Is this a biggy? I prefer using braces as it makes the code branch more visible at a glance. If I see it I then get an opportunity to try and remove it by simplifying the code